src/test/java/uk/gov/justice/probation/courtcasematcher/service/CourtCaseServiceTest.java

```diff
@@ -183,8 +183,7 @@ void givenMultipleSearchResponses_whenUpdateProbationStatus_thenIgnore() {
 
         var localDate = LocalDate.of(2020, Month.AUGUST, 20);
         var courtCase = CourtCase.builder().crn(CRN).courtCode(COURT_CODE).caseNo(CASE_NO)
-            .probationStatus("Previously known")
-            .probationStatusActual("PREVIOUSLY_KNOWN")
+            .probationStatus("PREVIOUSLY_KNOWN")
```

Comment:
I think this should be "Previously known"

Reply:
Re the value, this should be driven by the response from offender search shouldn't it? I'm expecting this will have the following [values](https://github.com/ministryofjustice/community-api/blob/089d7bd649db3b5be96acd20c59d39ea12c68d31/src/main/java/uk/gov/justice/digital/delius/data/api/ProbationStatus.java), ultimately defined in the community-api 🤔